### DISCUSSION TITLE
CodeQL paths/paths-ignore only for Python and JS?

### DISCUSSION THREAD

**Discussion by aleks-ivanov** [community/community#24984] [Actions]
<p>This is an odd decision.</p>
<p>Is there currently a way to allow those properties to work for “csharp” for example ?</p>

**Comment by criemen** [comment, marked as answer]
<p>Hi <a class="mention" href="/aleksivanovsinglet">@AleksIvanovSinglet</a> ,</p>
<p>I am sorry about the late reply.<br>
The team is discussing extending the paths-ignore feature beyond python and javascript.<br>
Unfortunately, I don’t have a timeline I could share at the moment.</p>
